frontend/include/chpl/uast/PragmaList.h

```diff
@@ -585,7 +585,6 @@ PRAGMA(CG_REPRESENTATIVE, npr, "cg representative", ncm)
 // this instantiation does not need to be resolved
 PRAGMA(CG_INTERIM_INST, npr, "cg interim instantiation", ncm)
 
-PRAGMA(SINGLE, ypr, "single", ncm)
```

Comment:
This is what gets rid of FLAG_SINGLE, right?  My knowledge of the new compiler is not as good as it could be

Reply:
Yes, this is `FLAG_SINGLE`